src/google-drive/credentials.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"915832868969-c9v2uoc5465vvc8ete678iqijnijv900.apps.googleusercontent.com","project_id":"timesheettest-181019","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://accounts.google.com/o/oauth2/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"373AqulqHmgnk4ojDK5sfNly","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
We don't want to check in secrets. We'll have to shove this into the `.env` file or come up with a better plan for dealing with this.